### ISSUE TITLE
June focus group testing: Improve the list of accessibility criteria on the user Profile

### ISSUE THREAD

**Issue by Blandine-AA** [hhaccessibility/hhaccessibility.github.io#610]
Users are complaining that the list of accessibility  criteria is too long. There are 58 questions. How do we make it more user friendly?

**Comment by M4ttoF** [comment]
We could try sorting based on type of disability. That way can easily find what pertains to them

**Comment by joshi1983** [comment]
There is a feature mentioned in #502 that could be useful for this.  It involves conditionally showing/hiding questions if related questions are answered a certain way.

**Comment by joshi1983** [comment]
#656 might help with this.  The shorter wording should make the list look smaller.

**Comment by joshi1983** [comment]
I like Matthew's suggestion with organizing them by disability.

vision, mobility, cognitive, other may be good disability-related labels to start with.

**Comment by stephanosm** [comment]
I agree. There should be a single question as to where a user "belongs" with pre-checked criteria. Once a user makes his selection then all related criteria appear underneath with a message something like "Please ensure the below criteria apply to you" - showing all criteria. Then next to this message a message something like "Not now". This would be for users who are in a hurry to go through the list of criteria.
